lib/ddr/contacts.rb

```diff
@@ -0,0 +1,27 @@
+module Ddr
+  module Contacts
+    extend ActiveSupport::Autoload
+
+    autoload :Contact
+
+    @@contacts = {}
+
+    def self.get(contact_slug)
+      @@contacts[contact_slug]
+    end
+
+    def self.contacts
+      @@contacts
+    end
+
+    def self.load_contacts
+      contacts_file = File.join(Rails.root, 'config', 'contacts.yml')
+      YAML.load_file(contacts_file).each do |key, value|
+        @@contacts[key] = Contact.new(value.merge('slug' => key))
```

Comment:
I would use an OpenStruct here (http://ruby-doc.org/stdlib-2.2.2/libdoc/ostruct/rdoc/OpenStruct.html) instead of Hash to avoid the metaprogramming in Contact.